docs-source/content/userguide/managing-domains/choosing-a-model/_index.md

```diff
@@ -5,25 +5,25 @@ weight = 1
-| Logs are automatically placed on persistent storage and sent to the pod's stdout.  | Logs are kept in the images and sent to the pod's log (`stdout`) by default. To change their location, you can set the domain resource `logHomeEnabled` to true and configure the desired directory using `logHome`. | Same as Domain in Image.  |
+| Lets you use the same standard WebLogic Docker image for every server in every domain. | Requires a different image for each domain, but all servers in that domain use the same image. | Different domains can use the same image, but require different domainUID and may have different configuration.  |
+| No state is kept in Docker images making the containers created from these images completely throw away (cattle not pets). | Runtime state should not be kept in the images, but applications and configuration are. | Runtime state should not be kept in the images.  Application and configuration may be. |
+| The domain is long-lived, so you can mutate the configuration or deploy new applications using the Administration Console or WLST. You can also mutate the configuration using configuration overrides. | If you want to mutate the domain home configuration, then you can apply configuration overrides or create a new image. If you want to deploy application updates, then you must create a new image. | If you want to mutate the domain home configuration, then you can override it with additional model files supplied in a ConfigMap or you can supply a new image. If you want to deploy application updates, then you must create a new image.  |
+| You can use configuration overrides to mutate the domain configuration, but there are [limitations]({{< relref "/userguide/managing-domains/configoverrides/_index.md#unsupported-overrides" >}}) and non-dynamic configuration attributes can only be changed when servers are starting. | You can use configuration overrides to mutate the domain configuration, but there are [limitations]({{< relref "/userguide/managing-domains/configoverrides/_index.md#unsupported-overrides" >}}) and non-dynamic configuration attributes can only be changed when servers are starting. You should not use the Administration Console or WLST for these domains as changes are ephemeral and will be lost when servers restart. | You can deploy model files to a ConfigMap to mutate the domain, and may not need to restart the entire domain for the change to take effect. Instead, you can initiate a rolling upgrade, which restarts your WebLogic Server instance Pods one at a time. Also, the model file syntax is far simpler and less error prone than the configuration override syntax, and, unlike configuration overrides, allows you to directly add data sources and JMS modules. You should not use the Administration Console or WLST for these domains as changes are ephemeral and will be lost when servers restart. |
```

Comment:
The limitations link here points into the config-overrides documentation, which is out of date as it assumes old behavior for override distribution.  The limitations section should be reworked to discuss the new capabilities and point to the new documentation about same.  (Overall the config-overrides doc looks like it's 'check-list', 'step-by-step', and 'limitations' sections still could use updates in this area...)

Reply:
I'm not quite sure what you want me to change... The link is to one of the other documents in this PR that is updated.